### ISSUE TITLE
Inconsistent White Space

### ISSUE THREAD

**Issue by jwrb** [PaycoinFoundation/paycoin#394]
Many files across the repo have inconsistent white space at the top of the file. This is usually under the copyright notices.

We need to have a discussion about whether we want to remove this white space, or include it in every file (for the sake of consistency).

This issue was carried down from Peercoin.

cc: @IngCr3at1on @ligerzero459 @mitchellcash

**Comment by ligerzero459** [comment]
I vote for including the whitespace in every file. I believe it looks cleaner that way

**Comment by IngCr3at1on** [comment]
As I mentioned in chat to @jwrb I have no strong opinion on this one way or the other but I do agree consistency would be a great idea.

**Comment by mitchellcash** [comment]
Agree on consistency and side with @ligerzero459 to include the whitespace as it makes the code look cleaner.

Now to watch someone implement this commit and break everyone's build branches when it's merged :P

**Comment by IngCr3at1on** [comment]
Hahaha, totally @mitchellcash

**Comment by jwrb** [comment]
My personal opinion is that whitespace at the top looks better.
This would however have to be a slowly incremented change so we don't break all current PR's. Perhaps we could set some guidelines for formatting? All future PR's follow them and we can slowly work back through and sort formatting throughout the entire codebase?

**Comment by IngCr3at1on** [comment]
IMHO even if it does break all outstanding PRs it's not really THAT big of an issue, any conflicts it creates will be extremely easy to resolve and being that we rarely touch the includes or copyright information it's really unlikely that it will break that much.

**Comment by mitchellcash** [comment]
Yeah I don't actually mind if you do a massive breaking change. Will be some of the easiest conflicts to fix, if it even conflicts.

But yeah on the topic of styling I have my thoughts. I've actually got most of the work done in a local branch already using .clang-format (I've had it sitting there for a while now so I will have to dust it off again if we are having this discussion)

**Comment by IngCr3at1on** [comment]
I was just looking at .clang-format yesterday.

**Comment by mitchellcash** [comment]
What did you think? I can get a pull in

**Comment by IngCr3at1on** [comment]
I have nothing against .clang-format but since it's a formatting tool I obviously wouldn't be able to say if I agree with the formatting you implemented without seeing it :P (strictly speaking I hate most of the formatting on this repo I just ignore it cause it's a hodgepodge random bits slapped together with no regards for formatting in the first place).

A formatting PR is a good place to discuss said styling though so yes, go ahead if you like.

**Comment by mitchellcash** [comment]
Done, see #404
